code/rounds/MapSelectionRound.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Sandbox;
+
+using TTTReborn.Globals;
+using TTTReborn.Player;
+using TTTReborn.Settings;
+using TTTReborn.UI;
+
+namespace TTTReborn.Rounds
+{
+    public class MapSelectionRound : BaseRound
+    {
+        public override string RoundName => "Map Selection";
+        public override int RoundDuration
+        {
+            get => ServerSettings.Instance.Round.MapSelectionRoundTime;
+        }
+
+        protected override void OnTimeUp()
+        {
+            base.OnTimeUp();
+
+            IDictionary<int, int> mapIndexToVoteCount = MapSelection.GetTotalVotesPerMapIndex(Gamemode.Game.Instance.NextMapVotes);
+            if (mapIndexToVoteCount.Count == 0)
+            {
+                Global.ChangeLevel("facepunch.flatgrass");
```

Comment:
It wold be awesome if this would be configurable to a custom default/fallback map.

Reply:
Yeah that was the plan 👍